oas_docs/README.md

```diff
@@ -1,9 +1,164 @@
+
+## Bundling commands
+
+Scripts in the `scripts` folder bundle individual Kibana OpenAPI bundles. There is a makefile to simplify the workflow. The following makefile targets are available
+
+| Command                    | Description                                                                                                                                                                                                        |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `api-docs`                 | Builds ESS Kibana OpenAPI bundle                                                                                                                                                                                   |
+| `api-docs-serverless`      | Builds Serverless Kibana OpenAPI bundle                                                                                                                                                                            |
+| `api-docs-lint`            | Lints built result bundles                                                                                                                                                                                         |
+| `api-docs-lint-errs`       | Lints built result bundles for errors                                                                                                                                                                              |
+| `api-docs-preview`         | Generates (ESS + Serverless) Kibana OpenAPI bundles preview                                                                                                                                                        |
+| `api-docs-overlay`         | Applies [overlays](https://docs.bump.sh/help/specification-support/overlays/) from `overlays` folder to the Kibana OpenAPI bundles and generate `*.new.yaml` files. Overlays help to fine tune the result bundles. |
+| `api-docs-overlay-preview` | Generates a preview for bundles produced by `api-docs-overlay`                                                                                                                                                     |
+
+## Bundling automation
+
+To have always up to date Kibana OpenAPI bundles bundling runs as a part of `Checks` step in CI on every PR build and on merge builds. If there are any changes in source OpenAPI files these changes get to the updated Kibana OpenAPI bundles and CI commits the changes. In that case CI is marked as failed and needs to be restarted.
+
+## Bundling solution domains
+
+### Security Solution
```

Comment:
Besides paths to the bundling scripts and output folders, can we please elaborate on how these scripts are integrated into the CI?

Reply:
I've added a new sub section but it basically very similar to `Bundling automation` section.